setup.py

```diff
@@ -206,17 +206,18 @@ def run(self):
     'pyqt5<5.13',
     'pyqtwebengine<5.13',
     'python-language-server[all]>=0.35.0,<1.0.0',
-    'pyls-black',
+    'pyls-black>=0.4.6',
     'pyxdg>=0.26;platform_system=="Linux"',
     'pyzmq>=17',
     'qdarkstyle>=2.8',
     'qtawesome>=0.5.7',
     'qtconsole>=4.7.7',
     'qtpy>=1.5.0',
+    'setuptools>=39.0.0',
```

Comment:
So i guess `pkg_resources` is a dependency of `setuptools` since this version, right? Just checking with you.

Reply:
Correct. Not sure which version is actually required, but saw some Stack exchange answer where 39 was suggested, but more or a guess.

Reply:
Ok, good to know. Thanks for digging into that.